Handler/TagHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FOS\HttpCacheBundle\Handler;
+
+use FOS\HttpCache\Handler\TagHandler as BaseTagHandler;
+use Symfony\Component\HttpFoundation\Response;
+
+class TagHandler extends BaseTagHandler
+{
+    /**
+     * Tag response with the previously added tags.
+     *
+     * @param Response $response
+     * @param bool     $replace  Whether to replace the current tags on the
+     *                           response
+     *
+     * @return $this
+     */
+    public function tagResponse(Response $response, $replace = false)
+    {
+        if (!$replace && $response->headers->has($this->getTagsHeaderName())) {
+            $header = $response->headers->get($this->getTagsHeaderName());
+            if ('' !== $header) {
+                $this->addTags(explode(',', $response->headers->get($this->getTagsHeaderName())));
+            }
+        }
+
+        if ($this->getTagsHeaderValue()) {
```

Comment:
created https://github.com/FriendsOfSymfony/FOSHttpCache/pull/190

we will need to tag 1.3.2 afterwards and this PR depends on 1.3.2 then.